### ISSUE TITLE
Allow overriding :GetSequenceGroundSpeed() for SNPCs

### ISSUE THREAD

**Issue by MerekiDor** [Facepunch/garrysmod-requests#2531]
There is no obvious way to alter a custom NPC's ground speed. At least, not to my knowledge. It currently depends on the information stored in the model's sequence.

Please let us override `CBaseAnimating::GetSequenceGroundSpeed()` (_source-sdk-2013/mp/src/game/server/baseanimating.cpp_) of lua NPCs, or at least provide a way to set it to a fixed value (`NPC:SetIdealMoveSpeed()`?) so that we can make NPCs with any model move without resorting to setting up a custom `ENTITY:OverrideMove()`.

**Comment by BREEDNKMs** [comment]
i agree, this would make snpcs making process a lot easier and faster. for a long time, we should have had a way to return a move speed for snpcs in case they have models with no root bone animation.

**Comment by BREEDNKMs** [comment]
bump
